### PR TITLE
Add LAHS first meeting planning.md & recap.md

### PR DESCRIPTION
**Pull request by MaxWofford**
_not ready for merge_

### CODE REVIEW

File: case_studies/los_altos_high_school/15_09_11_first_meeting/recap.md [hunk 1]
```diff
@@ -1 +1,49 @@
 # Recap
+
+## Summary
+
+- 13 people showed up due to technical difficulties
+- Went over what a hacker was - someone who builds, makes, creates
+  - Talented individuals who can design code and ship amazing products
+- Our d2f - dream to fulfill
```
Comment (MaxWofford) [inline case_studies/los_altos_high_school/15_09_11_first_meeting/recap.md]: > Our d2f

This sounds  like a fun activity. What does it entail?

File: case_studies/los_altos_high_school/15_09_11_first_meeting/recap.md [hunk 2]
```diff
@@ -1 +1,49 @@
 # Recap
+
+## Summary
+
+- 13 people showed up due to technical difficulties
```
Comment (MaxWofford) [inline case_studies/los_altos_high_school/15_09_11_first_meeting/recap.md]: What were these technical difficulties? Is it possible for other clubs to run into these issues?

**Comment by TheThirdOne** [comment]
> d2f

That immediately brought [silicon valley](https://www.youtube.com/watch?v=Tx3wDTzqDTs) to mind.

Probably not a big deal, but you may want to reconsider that acronym.

**Comment by paked** [comment]
@TheThirdOne I agree

```
...
```



I'M SORRY I COULDN'T HOLD IT BACK

**Pull request merged**